aclk/aclk_common.c

```diff
@@ -42,7 +42,7 @@ ACLK_PROXY_TYPE aclk_verify_proxy(const char *string)
     if (!string)
         return PROXY_TYPE_UNKNOWN;
 
-    while (*string == 0x20)
+    while (*string == 0x20 && *string!=0)   // Help coverity (compiler will remove)
```

Comment:
I'd better mark this as a false positive.

Reply:
I considered that, but the code may change in the future and making Coverity understand that the input is sanitized is more robust.

Reply:
Maybe. But it looks ugly :).